src/main/helm/helm-service.ts

```diff
@@ -3,29 +3,27 @@ import logger from "../logger";
 import { repoManager } from "./helm-repo-manager";
 import { HelmChartManager } from "./helm-chart-manager";
 import { releaseManager } from "./helm-release-manager";
+import { HelmChart } from "../../renderer/api/endpoints/helm-charts.api";
+
+import type { HelmChartList, RepoHelmChartList } from "../../renderer/api/endpoints/helm-charts.api";
```

Comment:
You can `import` types from line above since we're importing `HelmChart` class as well.